app/services/coffee_shop_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Services
+  class CoffeeShopService
+    include Helpers::DistanceHelper
+
+    def initialize(user_coords, coffee_shops)
+      @user_coords = user_coords
+      @coffee_shops = coffee_shops
+    end
+
+    def call
+      closest_shops(calculate_distances_hash)
+    end
```

Comment:
the API of this command doesn't tell what it does

Reply:
Yes, a better name could be `CoffeeShopLocatorService`.